rosgraph_msgs/TopicStatistics[] topics
bool buffering
